core/lib/types/src/snapshots.rs

```diff
@@ -172,3 +172,14 @@ impl ProtoFmt for SnapshotStorageLogsChunk {
         }
     }
 }
+
+#[derive(Debug, Serialize, Deserialize, PartialEq)]
```

Comment:
Nit: If we don't need `Serialize` and `Deserialize` rn, it makes sense to not derive them; these traits could trivially be derived later if necessary.

Reply:
done